Makefile

```diff
@@ -18,4 +30,21 @@ py:
 
 get_com:
 	source ${VENV_DIR}/bin/activate; \
-	python scripts/download_from_lesswrong.com.py > log.txt 2>&1
+	python scripts/download_from_lesswrong.com.py > $(LOG_FILE) 2>&1
+
+book:
+	source ${VENV_DIR}/bin/activate; \
+	python scripts/html_to_docbook.py > $(LOG_FILE) 2>&1
```

Comment:
Make sense to validate result like I do [here](https://github.com/worldmind/ratiocracy.book/blob/main/Makefile#L80).